zerver/lib/actions.py

```diff
@@ -3417,6 +3417,9 @@ def check_message(
             # else can sneak past the access check.
             assert sender.bot_type == sender.OUTGOING_WEBHOOK_BOT
 
+        if realm.mandatory_topics and topic_name == "(no topic)":
+            raise JsonableError(_("Require topics in stream messages."))
```

Comment:
This error message should be tweaked to explain what's happening.  I think we want "Topics are required in this organization".  @alya FYI.

Reply:
Sure, that seems fine. While we're at it, we should replace the other error message ("Please specify a topic") with the same sentence.